src/Util/PHP/Template/TestCaseMethod.tpl

```diff
@@ -122,5 +122,14 @@ if (isset($GLOBALS['__PHPUNIT_BOOTSTRAP'])) {
     require_once $GLOBALS['__PHPUNIT_BOOTSTRAP'];
     unset($GLOBALS['__PHPUNIT_BOOTSTRAP']);
 }
+var_dump( $GLOBALS );
+if (isset($GLOBALS['__PHPUNIT_TESTSUITE_BOOTSTRAPS'])) {
+    var_dump( $GLOBALS['__PHPUNIT_TESTSUITE_BOOTSTRAPS']);
```

Comment:
This looks like a leftover from debugging.

Reply:
Good eye. Missed those.